wp-config.php

```diff
@@ -88,7 +88,11 @@
     if ( in_array( $_ENV['PANTHEON_ENVIRONMENT'], array( 'test', 'live' ) ) && ! defined( 'DISALLOW_FILE_MODS' ) ) :
         define( 'DISALLOW_FILE_MODS', true );
     endif;
-
+    
+    if (file_exists(dirname(__FILE__) . '/wp-config-pantheon.php')) :
```

Comment:
As long as we are touching wp-config.php, let's move all of the Panthon-specific changes to the wp-config-panthon.php file. That way, if we have future need to change any Pantheon-specific behavior, we can do it without touching wp-config.php.

Reply:
Is this introducing a new file named `wp-config-panthon.php`? Doesn't that open the door for moving all non-WordPress Core wp-config modifications to that file? Probably everything under the comment `/** A couple extra tweaks to help things run well on Pantheon. **/`.

Reply:
Yes, that was my suggestion exactly.